apps/web/app/WithEmbedSSR.tsx

```diff
@@ -16,7 +17,8 @@ export default function withEmbedSsrAppDir<T extends Record<string, any>>(
     const { embed, layout } = context.query;
     const isCOEPEnabled = context.query["flag.coep"] === "true";
     if (isCOEPEnabled) {
-      context.res.setHeader("Cross-Origin-Embedder-Policy", "require-corp");
+      const response = NextResponse.next();
+      response.headers.set("Cross-Origin-Embedder-Policy", "require-corp");
```

Comment:
Why did you changed this?

Reply:
I'll wait for @hariombalhara approval on this one

Reply:
`context.res.setHeader` will throw an error in app router.

https://github.com/calcom/cal.com/blob/1edd2558466d98f0768f2553a1324589daee9ccd/apps/web/lib/buildLegacyCtx.tsx#L39-L44

Reply:
This newly created response variable isn't being used anywhere. So, this effectively makes it not work at all.

Have fixed it here
https://github.com/calcom/cal.com/pull/16453#discussion_r1745010098